index.html

```diff
@@ -2455,113 +2455,129 @@ <h2>Services</h2>
-into the <a href="#data-model">data model</a> and then serializing the <a
-href="#data-model">data model</a> into the target <a>representation</a>.
+A concrete serialization of any data structure in this specification is called a
+<a>representation</a>. A <a>representation</a> is created by serializing the <a
+href="#data-model">data model</a> through a process called
+<dfn>production</dfn>. A <a>representation</a> is transformed back into the <a
+href="#data-model">data model</a> through a process called
+<dfn>consumption</dfn>. The <em>production</em> and <em>consumption</em> process
+enables the conversion of information from one <a>representation</a> to another.
+Although this specification defines <a>representations</a> for JSON, JSON-LD,
+and CBOR, developers can use any other <a>representation</a>, such as XML or
+YAML, that is capable of expressing the <a href="#data-model">data model</a>.
+The following sections define the general rules for <a>production</a> and
+<a>consumption</a> and the JSON, JSON-LD, and CBOR <a>representations</a>.
     </p>
 
-    <p>
-Although syntactic mappings are provided for JSON, JSON-LD, and CBOR here,
-applications and services MAY use any other data <a>representation</a> syntax that is
-capable of expressing the <a href="#data-model">data model</a>, such as XML or
-YAML.
-    </p>
-
-    <p>
-Producers MUST indicate which <a>representation</a> of a document has been used via a
-media type in the document's metadata. Consumers MUST determine the
-<a>representation</a> of a <a>DID document</a> via the <code>contentType</code> <a>DID
-resolver</a> metadata field (see <a href="#did-resolution"></a>), <em>not</em>
-through the content of the <a>DID document</a> alone.
-    </p>
+    <section>
+      <h2>Production and Consumption</h2>
 
-    <p>
-Unrecognized entries in the <a href="#data-model">data model</a> MUST be
-preserved. An unrecognized entry is any entry that does not have explicit
-processing rules known to the consumer or producer. Consumers MUST add all
-entries that do not have explicit processing rules for the
-<a>representation</a> being consumed to the <a href="#data-model">data
-model</a> using only the <a>representation</a>'s generic type processing rules.
-Producers MUST serialize all entries in the <a href="#data-model">data
-model</a> that do not have explicit processing rules for the
-<a>representation</a> being produced using only the <a>representation</a>'s
-generic type processing rules.
-    </p>
+      <p>
+The rules in this section apply to all implementations seeking to be compatible
+with independent implementations of the specification. Deployments of this
+specification can use a custom agreed-upon <a>representation</a>, including
+rules agreed to by a pair of producers and consumers for handling properties not
+listed in the registry. See <a href="#extensibility"></a> for more information.
```

Comment:
```suggestion
As well as the <a>representations</a> defined in this specification,
implementers can use other <a>representations</a>, providing each
<a>representation</a> is properly specified (including
rules for handling properties not listed in the DID Specification
Registries [[?DID-SPEC-REGISTRIES]]). See <a
href="#extensibility"></a> for more information.
```

I'm having trouble parsing this. I've tried to rephrase, but not sure if I've captured what you meant.

Reply:
Maybe...?
```suggestion
In addition to the <a>representations</a> defined in this specification,
implementers can use other <a>representations</a>, providing each such
<a>representation</a> is properly specified (including rules for
interoperable handling of properties not listed in the DID Specification
Registries [[?DID-SPEC-REGISTRIES]]). See <a href="#extensibility"></a>
for more information.
```

Reply:
Applied @TallTed's suggestion.